library/Mockery.php

```diff
@@ -662,10 +662,22 @@ private static function objectToArray($object, $nesting = 3)
             return array('...');
         }
 
-        return array(
-            'class' => get_class($object),
-            'properties' => self::extractInstancePublicProperties($object, $nesting)
+        $defaultFormatter = function ($object, $nesting) {
+            return array('properties' => self::extractInstancePublicProperties($object, $nesting));
+        };
+
+        $class = get_class($object);
+
+        $formatter = self::getConfiguration()->getObjectFormatter($class, $defaultFormatter);
+
+        $array = array(
+          'class' => $class,
+          'identity' => '#' . md5(spl_object_hash($object))
```

Comment:
why?

Reply:
It is often useful to be able to compare object identities not only their class + some optional custom info from formatter. Should I remove it?

Reply:
I mean why the md5. Just use `spl_object_id`.

Reply:
Besause spl_object_id is hard to compare visually becaue usually two instances have 95% characters of spl_object_hash identical. Using of md5 of spl_object_hash is quite common practice i saw in many libraries.

Reply:
For example I just used it on two objects and I got:
000000002f73941b000000004f808351 and 000000002ffa4e380000000026e9a821
If you use md5 it becomes different at the first sight:
69850868ab17d5cd4d7effccd55ba63c and 43147e68957af1fcbd045979c3c3fda3